DuckDuckGo/TabSwitcherViewController+KeyCommands.swift

```diff
@@ -26,19 +26,30 @@ extension TabSwitcherViewController {
             
             UIKeyCommand(input: UIKeyCommand.inputEscape, modifierFlags: [], action: #selector(keyboardCloseWindow),
                          discoverabilityTitle: UserText.keyCommandClose),
+                        
+            UIKeyCommand(input: "w", modifierFlags: [.command], action: #selector(keyboardCloseWindow),
+                         discoverabilityTitle: UserText.keyCommandClose),
+
             UIKeyCommand(input: "t", modifierFlags: [ .command ], action: #selector(keyboardNewTab),
                         discoverabilityTitle: UserText.keyCommandNewTab),
             UIKeyCommand(input: "n", modifierFlags: [ .command ], action: #selector(keyboardNewTab),
                 discoverabilityTitle: UserText.keyCommandNewTab),
+            
             UIKeyCommand(input: UIKeyCommand.inputEnter, modifierFlags: [], action: #selector(keyboardSelectCurrent),
                          discoverabilityTitle: UserText.keyCommandSelect),
+            
             UIKeyCommand(input: UIKeyCommand.inputUpArrow, modifierFlags: [], action: #selector(keyboardMoveSelectionUp),
                          discoverabilityTitle: UserText.keyCommandPreviousTab),
             UIKeyCommand(input: UIKeyCommand.inputDownArrow, modifierFlags: [], action: #selector(keyboardMoveSelectionDown),
                          discoverabilityTitle: UserText.keyCommandNextTab),
+            
+            UIKeyCommand(input: UIKeyCommand.inputLeftArrow, modifierFlags: [], action: #selector(keyboardMoveSelectionUp),
+                         discoverabilityTitle: UserText.keyCommandPreviousTab),
+            UIKeyCommand(input: UIKeyCommand.inputRightArrow, modifierFlags: [], action: #selector(keyboardMoveSelectionDown),
+                         discoverabilityTitle: UserText.keyCommandNextTab),
+
```

Comment:
This is actually pretty confusing on a grid. We prolly should allow for 2-dimensional navigation there.

Reply:
I thought about that but Safari does the the same doesn’t it?

Reply:
For me keyboard doesn't work at all with tab switcher on safari. But still, even if that were the case, I think we can do a better job at this than Safari does. :)

Reply:
I looked at this and it's pretty complex for something that's supposed to be a quick improvement.  Safari with an actual keyboard definitely handles arrows the same way.  I can't get the simulator to handle arrow keys in the Safari tab switch though.